def divideNos(a, b):
   return a/b
 
try:
   divideNos(10, 3)
except:
   print('some exception occured')
